include/seastar/websocket/server.hh

```diff
@@ -35,7 +35,7 @@
 
 namespace seastar::experimental::websocket {
 
-using handler_t = std::function<future<temporary_buffer<char>>(temporary_buffer<char>)>; 
+using handler_t = std::function<temporary_buffer<char>(temporary_buffer<char>)>;
```

Comment:
If I understand correctly, you could have fixed this line in the previous patch already?